static/js/global.js

```diff
@@ -336,17 +338,17 @@ let metRatesTypicalTasks = [
         activity: 'Wrestling: 7.8',
         met: 7.8
     }];
-metRatesTypicalTasks.sort((a, b) => parseFloat(a.met) - parseFloat(b.met));
+// metRatesTypicalTasks.sort((a, b) => parseFloat(a.met) - parseFloat(b.met));
 
 // Metabolic rates of typical activities ASHRAE removed reclining and sleeping
 let metRatesTypicalTasksASHRAE = metRatesTypicalTasks.filter(function (el) {
     return el.met > 0.7 && // 0.8 is the reclining met
         el.met <= 4.0  // as defined in page 36 ASHRAE 55
 });
-metRatesTypicalTasksASHRAE.sort((a, b) => parseFloat(a.met) - parseFloat(b.met));
+// metRatesTypicalTasksASHRAE.sort((a, b) => parseFloat(a.met) - parseFloat(b.met));
```

Comment:
Do we not need these sort statements? I thought they were necessary for the tasks/activities to show up in order. We could sort them manually in the code instead. It's OK to remove code if it's not needed anymore, no need to comment it out.

Reply:
Yes you are right we should sort them manually since that function is not working in IE. I will remove them permanently from the code and sort the values manually.